scripts/run-integration-tests.sh

```diff
@@ -51,7 +51,7 @@ sleep 3
 
 # run tests
 echo "running integration tests..."
-TESTS=integration go test ./tests -v
+TESTS=integration go test ./tests -v -count=1
```

Comment:
Definitely in the integration tests, where there can be race conditions, we want to rerun all tests and not use cached results.  I made the same change in the unit tests script, but I'm happy to revert that change if we want faster runs of those tests.